astropy/time/core.py

```diff
@@ -1292,11 +1294,11 @@ def sidereal_time(self, kind, longitude=None, model=None):
             longitude = self.location.lon
         elif longitude == 'greenwich':
             longitude = Longitude(0., u.degree,
-                                  wrap_angle=180.*u.degree)
+                                  wrap_angle=180. * u.degree)
```

Comment:
Here, I don't like the spaces around the operator, given that it is argument assignment. Note that for binary operators the pep8 recommendations are not particularly strong: https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
I thought some of this might be slightly controversial.  :smile:  I guess for me, *most* of the time I find the code to be more readable if there are always spaces between operators, and so I just get in the habit of automatically putting in a space.  Then I stop having to spend brainpower thinking about precedence or context (kwarg vs not?) or details of the particular situation.

Then I find things like this weird:
```
x = y * z  # OK
x = y * z + 1  # Not OK?
```
And going back to the genius of formatting Knuth, in LaTeX `y = 1*2 + 3*x` gets rendered as:
![image](https://user-images.githubusercontent.com/348089/70837856-c38ed480-1dc2-11ea-9e20-0eeb44b4d1cc.png)
Likewise `c = (a+b) * (a-b)` in LaTeX looks basically like `c = (a + b) * (a - b)`.  So I vote for Knuth over the PEP8 authors.

Anyway we likely won't ever get complete agreement because what is "best" is a matter of taste and relates to how one's brain parses text.  As a practical matter, it is possible to turn on checking for spaces and then `# noqa` the cases where clarity is much improved by removing spaces.  The converse is not true as far as I know (could be wrong??), namely checking for rules based on precedence and other more subtle elements of context.

Reply:
But anyway, I'll revert some of the changes you mentioned and turn off that check in my astropy flake8 settings.